sdk/cosmosdb/cosmos/consumer-test.js

```diff
@@ -1,34 +0,0 @@
-const execa = require("execa");
-let tsVersionsToCheckCompatibility = ["4.2"];
```

Comment:
@mpodwysocki why are removing minimum typescript version check?

Reply:
We already have a policy in place that we do not support older versions of TypeScript that are outside of the [DefinitelyTyped Support Window](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#support-window).